lib/child-writer/writer-proxy.js

```diff
@@ -89,7 +89,7 @@ return permissions.isElevated().then((elevated) => {
     console.log(`Running: ${commandArguments.join(' ')}`);
 
     return permissions.elevateCommand(commandArguments, {
-      applicationName: packageJSON.build.productName,
+      applicationName: packageJSON.displayName,
```

Comment:
Like one of your comments in #1438 , should this be picking up the `initialTitle` rather than a value from `package.json` ?